src/main/java/nextstep/ladder/domain/LadderNames.java

```diff
@@ -0,0 +1,63 @@
+package nextstep.ladder.domain;
+
+import nextstep.ladder.exception.InputNullException;
+import nextstep.ladder.exception.StringLengthException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class LadderNames {
+    private static final int LETTER_LIMIT = 5;
+
+    private List<LadderName> ladderNames;
+
+    public static LadderNames make(String ladderName) {
+        return new LadderNames(ladderName);
+    }
+
+    private LadderNames(final String ladderNames) {
+        inputValidation(ladderNames);
+        List<LadderName> list = new ArrayList<>();
+        Arrays.stream(ladderNames.split(","))
+                .map(this::lengthValidation)
+                .forEach(name -> list.add(new LadderName(name)));
+        this.ladderNames = list;
+    }
```

Comment:
```suggestion
    private LadderNames(List<LadderName> ladderNames) {
        this.ladderNames = ladderNames;
    }
    
    public static LadderNames of(List<LadderName> ladderNames) {
        return new LadderNames(ladderName);
    }

    public static LadderNames of(String ladderName) {
        inputValidation(ladderNames);
        return Arrays.stream(ladderNames.split(","))
                .map(LadderName::valueOf) // ladderName도 valueOf나 of 같은 메소드로 객체를 생성할 수 있게 해주세요
                .collect(collectingAndThen(toList(), LadderNames::of));
    }
```
- 이렇게 String으로도 input을 받을 수 있고, List로도 input을 받을 수 있는 형태로 만들어주면 좋을 것 같아요 
- Stream을 잘못된 방식으로 사용중입니다. map - collect 등을 잘 활용해주세요